ddtrace/contrib/trace_utils.py

```diff
@@ -330,9 +330,38 @@ def activate_distributed_headers(tracer, int_config=None, request_headers=None,
 
     if override or (int_config and distributed_tracing_enabled(int_config)):
         context = HTTPPropagator.extract(request_headers)
+
+        # Toggle for the old behavior
+        # As long as we extracted a valid context, activate it
+        if not config.propagation_skip_multiple_extract:
```

Comment:
should this be a public attribute on the config?

Reply:
if we ever need someone to enable it, it would be weird to have them enable a private config option.

I don't think I have a strong opinion, it just seems counter-intuitive to offer a way for customers to toggle it but not make it public (so maybe we don't just don't need it?)

Reply:
It seems to me like it should be ok to make it public, maybe we should add a deprecation warning to it if toggled to True so we can remove it 2.x?